jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/util/EnvUtilTest.java

```diff
@@ -387,7 +387,7 @@ public void testStoreTimestamp() throws IOException {
     @Test
     public void testLoadTimestampShouldLoadFromFile() throws Exception {
         // Given
-        final File file = new File(EnvUtilTest.class.getResource("/util/loadTimestamp.timestamp").getFile());
+        final File file = new File(EnvUtilTest.class.getResource("/util/loadTimestamp.timestamp").toURI());
```

Comment:
Why is this needed?

Reply:
```
C:\Users\mnurisan\00_BIN\jdk-17.0.1+12\bin\java.exe -ea -javaagent:C:\Users\mnurisan\.m2\repository/org/jmockit/jmockit/1.49/jmockit-1.49.jar -Didea.test.cyclic.buffer.size=1048576 -javaagent:C:\Users\mnurisan\00_BIN\ideaIU-2021.2.3\lib\idea_rt.jar=59982:C:\Users\mnurisan\00_BIN\ideaIU-2021.2.3\bin -Dfile.encoding=UTF-8 -classpath C:\Users\mnurisan\00_BIN\ideaIU-2021.2.3\lib\idea_rt.jar;C:\Users\mnurisan\00_BIN\ideaIU-2021.2.3\plugins\junit\lib\junit5-rt.jar;C:\Users\mnurisan\00_BIN\ideaIU-2021.2.3\plugins\junit\lib\junit-rt.jar;C:\Users\mnurisan\00-PROJECTS\forks\jkube\jkube-kit\common\target\test-classes;C:\Users\mnurisan\00-PROJECTS\forks\jkube\jkube-kit\common\target\classes;C:\Users\mnurisan\.m2\repository\io\fabric8\kubernetes-client\5.11.2\kubernetes-client-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\kubernetes-model-core\5.11.2\kubernetes-model-core-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\kubernetes-model-common\5.11.2\kubernetes-model-common-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\kubernetes-model-rbac\5.11.2\kubernetes-model-rbac-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\kubernetes-model-admissionregistration\5.11.2\kubernetes-model-admissionregistration-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\kubernetes-model-apps\5.11.2\kubernetes-model-apps-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\kubernetes-model-autoscaling\5.11.2\kubernetes-model-autoscaling-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\kubernetes-model-apiextensions\5.11.2\kubernetes-model-apiextensions-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\kubernetes-model-batch\5.11.2\kubernetes-model-batch-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\kubernetes-model-certificates\5.11.2\kubernetes-model-certificates-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\kubernetes-model-coordination\5.11.2\kubernetes-model-coordination-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\kubernetes-model-discovery\5.11.2\kubernetes-model-discovery-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\kubernetes-model-events\5.11.2\kubernetes-model-events-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\kubernetes-model-extensions\5.11.2\kubernetes-model-extensions-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\kubernetes-model-flowcontrol\5.11.2\kubernetes-model-flowcontrol-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\kubernetes-model-networking\5.11.2\kubernetes-model-networking-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\kubernetes-model-metrics\5.11.2\kubernetes-model-metrics-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\kubernetes-model-policy\5.11.2\kubernetes-model-policy-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\kubernetes-model-scheduling\5.11.2\kubernetes-model-scheduling-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\kubernetes-model-storageclass\5.11.2\kubernetes-model-storageclass-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\kubernetes-model-node\5.11.2\kubernetes-model-node-5.11.2.jar;C:\Users\mnurisan\.m2\repository\com\squareup\okhttp3\okhttp\3.12.12\okhttp-3.12.12.jar;C:\Users\mnurisan\.m2\repository\com\squareup\okio\okio\1.15.0\okio-1.15.0.jar;C:\Users\mnurisan\.m2\repository\com\squareup\okhttp3\logging-interceptor\3.12.12\logging-interceptor-3.12.12.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\zjsonpatch\0.3.0\zjsonpatch-0.3.0.jar;C:\Users\mnurisan\.m2\repository\com\github\mifmif\generex\1.0.2\generex-1.0.2.jar;C:\Users\mnurisan\.m2\repository\dk\brics\automaton\automaton\1.11-8\automaton-1.11-8.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\openshift-client\5.11.2\openshift-client-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\openshift-model\5.11.2\openshift-model-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\openshift-model-clusterautoscaling\5.11.2\openshift-model-clusterautoscaling-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\openshift-model-operator\5.11.2\openshift-model-operator-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\openshift-model-operatorhub\5.11.2\openshift-model-operatorhub-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\openshift-model-machine\5.11.2\openshift-model-machine-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\openshift-model-whereabouts\5.11.2\openshift-model-whereabouts-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\openshift-model-monitoring\5.11.2\openshift-model-monitoring-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\openshift-model-storageversionmigrator\5.11.2\openshift-model-storageversionmigrator-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\openshift-model-tuned\5.11.2\openshift-model-tuned-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\openshift-model-console\5.11.2\openshift-model-console-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\openshift-model-machineconfig\5.11.2\openshift-model-machineconfig-5.11.2.jar;C:\Users\mnurisan\.m2\repository\io\fabric8\openshift-model-miscellaneous\5.11.2\openshift-model-miscellaneous-5.11.2.jar;C:\Users\mnurisan\.m2\repository\org\apache\commons\commons-lang3\3.11\commons-lang3-3.11.jar;C:\Users\mnurisan\.m2\repository\org\apache\commons\commons-compress\1.21\commons-compress-1.21.jar;C:\Users\mnurisan\.m2\repository\commons-io\commons-io\2.8.0\commons-io-2.8.0.jar;C:\Users\mnurisan\.m2\repository\com\google\code\gson\gson\2.8.6\gson-2.8.6.jar;C:\Users\mnurisan\.m2\repository\com\google\guava\guava\30.1.1-jre\guava-30.1.1-jre.jar;C:\Users\mnurisan\.m2\repository\com\google\guava\failureaccess\1.0.1\failureaccess-1.0.1.jar;C:\Users\mnurisan\.m2\repository\com\google\guava\listenablefuture\9999.0-empty-to-avoid-conflict-with-guava\listenablefuture-9999.0-empty-to-avoid-conflict-with-guava.jar;C:\Users\mnurisan\.m2\repository\com\google\code\findbugs\jsr305\3.0.2\jsr305-3.0.2.jar;C:\Users\mnurisan\.m2\repository\org\checkerframework\checker-qual\3.8.0\checker-qual-3.8.0.jar;C:\Users\mnurisan\.m2\repository\com\google\errorprone\error_prone_annotations\2.5.1\error_prone_annotations-2.5.1.jar;C:\Users\mnurisan\.m2\repository\com\google\j2objc\j2objc-annotations\1.3\j2objc-annotations-1.3.jar;C:\Users\mnurisan\.m2\repository\com\fasterxml\jackson\core\jackson-core\2.12.4\jackson-core-2.12.4.jar;C:\Users\mnurisan\.m2\repository\com\fasterxml\jackson\core\jackson-databind\2.12.4\jackson-databind-2.12.4.jar;C:\Users\mnurisan\.m2\repository\com\fasterxml\jackson\core\jackson-annotations\2.12.4\jackson-annotations-2.12.4.jar;C:\Users\mnurisan\.m2\repository\com\fasterxml\jackson\dataformat\jackson-dataformat-yaml\2.12.4\jackson-dataformat-yaml-2.12.4.jar;C:\Users\mnurisan\.m2\repository\org\yaml\snakeyaml\1.27\snakeyaml-1.27.jar;C:\Users\mnurisan\.m2\repository\com\fasterxml\jackson\datatype\jackson-datatype-jsr310\2.12.4\jackson-datatype-jsr310-2.12.4.jar;C:\Users\mnurisan\.m2\repository\org\slf4j\slf4j-api\1.7.25\slf4j-api-1.7.25.jar;C:\Users\mnurisan\.m2\repository\org\eclipse\jgit\org.eclipse.jgit\5.12.0.202106070339-r\org.eclipse.jgit-5.12.0.202106070339-r.jar;C:\Users\mnurisan\.m2\repository\com\googlecode\javaewah\JavaEWAH\1.1.7\JavaEWAH-1.1.7.jar;C:\Users\mnurisan\.m2\repository\javax\validation\validation-api\2.0.1.Final\validation-api-2.0.1.Final.jar;C:\Users\mnurisan\.m2\repository\org\fusesource\jansi\jansi\2.3.3\jansi-2.3.3.jar;C:\Users\mnurisan\.m2\repository\org\javassist\javassist\3.20.0-GA\javassist-3.20.0-GA.jar;C:\Users\mnurisan\.m2\repository\org\projectlombok\lombok\1.18.22\lombok-1.18.22.jar;C:\Users\mnurisan\00-PROJECTS\forks\jkube\jkube-kit\common-test\target\classes;C:\Users\mnurisan\.m2\repository\junit\junit\4.13.1\junit-4.13.1.jar;C:\Users\mnurisan\.m2\repository\org\hamcrest\hamcrest-core\1.3\hamcrest-core-1.3.jar;C:\Users\mnurisan\.m2\repository\org\jmockit\jmockit\1.49\jmockit-1.49.jar;C:\Users\mnurisan\.m2\repository\org\assertj\assertj-core\3.18.0\assertj-core-3.18.0.jar;C:\Users\mnurisan\.m2\repository\org\hamcrest\hamcrest-all\1.3\hamcrest-all-1.3.jar;C:\Users\mnurisan\.m2\repository\com\networknt\json-schema-validator\1.0.53\json-schema-validator-1.0.53.jar com.intellij.rt.junit.JUnitStarter -ideVersion5 -junit4 org.eclipse.jkube.kit.common.util.EnvUtilTest,testLoadTimestampShouldLoadFromFile

Process finished with exit code 0
```

Reply:
In your path you don't  have spaces. I do :)
With spaces, `getFile` doesn't work properly